src/python/pants/option/global_options.py

```diff
@@ -874,11 +874,14 @@ def register_options(cls, register):
             "tags ('-' prefix).  Useful with ::, to find subsets of targets "
             "(e.g., integration tests.)",
         )
+        ci_var = "CI" in os.environ
```

Comment:
Nit: consider inlining this. It's simple enough that the variable adds more noise than benefit, IMO.